generator.yml

```diff
@@ -48,6 +49,7 @@ architectures:
 triples:
   - &arm-triple         {CROSS_COMPILE: arm-linux-gnueabi-,    ARCH: *arm-arch}
   - &arm64-triple       {CROSS_COMPILE: aarch64-linux-gnu-,    ARCH: *arm64-arch}
+  - &hexagon-triple     {CROSS_COMPILE: hexagon-linux-musl-,   ARCH: *hexagon-arch}
```

Comment:
I don't really understand the setup here, so I have little input to provide on the rest. But this bit was useful to me, thanks.